packages/kori-zod-openapi-plugin/src/index.ts

```diff
@@ -0,0 +1,15 @@
+import { defineKoriPlugin } from 'kori';
+
+export function zodOpenApiPlugin(): ReturnType<typeof defineKoriPlugin> {
+  return defineKoriPlugin({
+    name: 'zod-openapi',
+    version: '1.0.0',
```

Comment:
It would be better to use the same version as the package itself. How do you think?